app/templates/partials/stylesheets.jade

```diff
@@ -1,2 +1,2 @@
 link(rel='stylesheet', href='/styles/main.css')
-
+link(rel='stylesheet', href='/lib/horsey/dist/horsey.min.css')
```

Comment:
There's a way to link to a Sass-compiled CSS file without directly linking to it? Is that something server-side?

http://sass-lang.com/documentation/file.SASS_REFERENCE.html#custom_importers ?